WebRoot/css/zm.css

```diff
@@ -4021,23 +4021,69 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 /* ZmChooseFolderDialog and ZmPickTagDialog */
+
+#ChooseFolderDialog {
+	@ChooseFolderDialog@
+}
+
+#ChooseFolderDialog .WindowOuterContainer {
+	@ChooseFolderDialogContainer@
+}
+
 .ChooserDialog {
-	width:320px;
+	@ChooseFolderDialogBody@
 }
 
 .ChooserDialog .DwtInputField {
-	padding-bottom:3px;
-	padding-top:3px;
+	@ChooseFolderDialogInputContainer@
 }
 
-.ChooserDialog INPUT {
-	width:100%;
+#ChooseFolderDialog .ChooserDialog INPUT {
+	@ChooseFolderDialogInputField@
 }
 
 .ChooserDialog .overview {
-	background-color:white;
-	border:1px solid black;
-	overflow:auto;
+	@ChooseFolderDialogFolderContainer@
+}
+
+.ChooserDialog .dialogOverview {
+	@ChooseFolderDialogFolderContainerContent@
+}
+
+.ChooserDialog .DwtTreeItem {
+	@ChooseFolderDialogFolderItem@
+}
+
+.ChooserDialog .DwtTreeItemLevel1ChildDiv {
+	padding-left:0px;
+}
+
+.ChooserDialog .DwtTreeItemLevel1ChildDiv .DwtTreeItem-Control {
+	margin-top:11.5px;
+}
+
+.ChooserDialog svg.ImgContactsFolder {
+	fill:@IconChooseFolder@
+}
+
+.ChooserDialog .DwtTreeItem svg.ImgEmailedContacts{
+	fill:@IconChooseFolder@
+}
+
+.ChooserDialog .DwtTreeItem svg.ImgNodeExpanded {
+	fill:@SecondaryBrandingColor@
```

Comment:
Can you refer this:

```
.DwtTreeItem .svg-icon {
    fill: @TreeItemTextColor@
}
```

I guess you are trying to achieve same thing here, for consistency, can you reuse it?